Package.resolved

```diff
@@ -0,0 +1,16 @@
+{
```

Comment:
Do we even need this file? I think it's worth to delete / untrack it.

Reply:
I'd suggest keeping it. Similar to lock files in `yarn` and `npm`, this file helps keep every person who checks out this repo on the right version.

### Reference:
__(Coordinate Package Versions Across Your Team)__ https://developer.apple.com/documentation/xcode/adding_package_dependencies_to_your_app

Reply:
Okay, reasonable. At least, update CryptoSwift version and revision to fit the actual "1.3.2" one.